src/icrar/leap-accelerate/model/Integration.cc

```diff
@@ -21,26 +21,22 @@
  */
 
 #include "Integration.h"
+#include <icrar/leap-accelerate/math/linear_math_helper.h>
 #include <icrar/leap-accelerate/ms/utils.h>
 #include <icrar/leap-accelerate/ms/MeasurementSet.h>
 #include <icrar/leap-accelerate/common/Tensor3X.h>
 
 namespace icrar
 {
-    Integration::Integration(const casacore::MeasurementSet& ms, int integrationNumber, int channels, int baselines, int polarizations, int uvws)
+    Integration::Integration(const icrar::MeasurementSet& ms, int integrationNumber, int channels, int baselines, int polarizations, int uvws)
     : integration_number(integrationNumber)
     , index(0)
     , x(0)
     , channels(channels)
     , baselines(baselines)
     {
-        auto cms = icrar::MeasurementSet(ms);
-
-        data = cms.GetVis();
-        //uvw = cms.GetCoords(index); //TODO: convert to uvw
-
-        uvw = std::vector<casacore::MVuvw>();
-        uvw.resize(uvws);
+        data = ms.GetVis(channels, baselines, polarizations);
```

Comment:
It seems that I'm getting a different tensor from the MeasurementSet than the python implementation here:

LEAP-Cal
![image](https://user-images.githubusercontent.com/7478405/91187448-c2b88b80-e722-11ea-9545-1192a320a0ef.png)

leap-accelerate
![image](https://user-images.githubusercontent.com/7478405/91187531-dbc13c80-e722-11ea-9f00-d8277e944ea9.png)

Reply:
I'll need to create a unit test suite for the Integration class to rectify this as it is rather difficult to quickly print and check a 3D matrix on screen

Reply:
I remember that I added a test called `test_read_vis` but only read up to channel 2 instead of past 4 as shown above. I'll extend this test a bit further and compare with LEAP-Cal